src/api/app/helpers/webui/user_helper.rb

```diff
@@ -67,7 +67,11 @@ def user_with_realname_and_icon(user, opts = {})
         printed_name = "#{realname} (#{user.login})"
       end
 
-      user_image_tag(user) + ' ' + link_to(printed_name, user_show_path(user))
+      if opts[:no_icon]
+        link_to(printed_name, user_show_path(user))
+      else
+        "#{user_image_tag(user)} #{link_to(printed_name, user_show_path(user))}".html_safe
```

Comment:
@dmarcoux 

> I would interpolate this: `"#{user_image_tag(user)} #{link_to(printed_name, user_show_path(user))}"`

I disagree here... `link_to` is not returning an String (in which case interpolation would make sense) but an HTML tag (for which rails is already calling `html_safe`). Converting it to String by using interpolation and calling `html_safe` again is IMO not an improvement.

Reply:
@Ana06 that makes sense, i'm going to revert to earlier state